components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/modules/oauth/token-protected-service-invokers.js

```diff
@@ -116,6 +116,11 @@ var invokers = function () {
         log.debug("Request payload if any : " + stringify(requestPayload));
         log.debug("Response status : " + xmlHttpRequest.status);
         log.debug("Response payload if any : " + xmlHttpRequest.responseText);
+        if (devicemgtProps["isCloud"]) {
```

Comment:
@rasika IMO these logs were introduced for auditing purposes and to speed up the debugging process.